src/PayU/Payments/Gateways/AluV3/Services/HashService.php

```diff
@@ -1,8 +1,11 @@
 <?php
 
-namespace PayU\Alu;
```

Comment:
BC break: we should not move/rename old classes, we should only mark them as deprecated.
I suggest we create new classes and have the old classes use them.

Reply:
done